source/Conventions.rst

```diff
@@ -25,7 +25,7 @@ Navigation
 User Interface & Configuration
 ==============================
 
-.. _ui-cli
+.. _ui-cli:
 
 Command Line Interface (CLI)
 ++++++++++++++++++++++++++++
```

Comment:
ln 37/41: we have a subcommand without a command? that gets muddy in the terminology

Reply:
In the terminology of the CLI library we use, and I think more generally, the "command" is the executable name.  I'll clarify that in the text.